<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_addonTag">
     <property name="text">
      <string>Addon Tag</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineEdit_addonTag"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_nameSpace">
     <property name="text">
      <string>Namespace</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="lineEdit_nameSpace"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_useAddonTag">
     <property name="text">
      <string>Use addon tag</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_terrainName">
     <property name="text">
      <string>Terrain name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_terrainName"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_terrainClassName">
     <property name="text">
      <string>Terrain class name</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_terrainClassName"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label_terrainSize">
     <property name="text">
      <string>Terrain size in meters</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QSpinBox" name="spinBox_terrainSize">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000000</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_author">
     <property name="text">
      <string>Author name (you)</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_author"/>
   </item>
   <item row="12" column="2">
    <widget class="QPushButton" name="pushButton_createConfig">
     <property name="text">
      <string>Create Config</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="checkBox_fromName">
     <property name="text">
      <string>From name</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
